tfs/src/main/java/hudson/plugins/tfs/util/BuildWorkspaceConfigurationRetriever.java

```diff
@@ -45,17 +45,23 @@ public BuildWorkspaceConfiguration getLatestForNode(Node needleNode, Run<?,?> la
     
     public static class BuildWorkspaceConfiguration extends WorkspaceConfiguration {
         private static final long serialVersionUID = 1L;
-        private final AbstractBuild<?, ?> build;
+        private final transient AbstractBuild<?, ?> build;

```

Comment:
I am not sure what is the use-case for this class, but it may also break persistency on the disk 🐜